### DISCUSSION TITLE
Welcome to Uzbekistan regions data Discussions!

### DISCUSSION THREAD

**Discussion by MIMAXUZ** [MIMAXUZ/uzbekistan-regions-data#1] [Ideas]
## 👋 Xush kelibsiz!
Biz munozaralardan jamoamizning boshqa a'zolari bilan bog'lanish uchun joy sifatida foydalanmoqdamiz. Umid qilamizki:
  * Sizni qiziqtirgan savollaringizni berasiz.
  * Fikrlar bilan o'rtoqlashasiz.
  * Jamoaning boshqa a'zolari bilan aloqada bo'lasiz.
  * Boshqalarni xush kelibsiz  deymiz va ochiq fikrlar tarafdorimiz. 
   Shuni unutmangki, bu biz birgalikda quradigan jamoa 💪.

Quyida o'zingizni qiziqtirgan firklar va takliflar haqida yozib qoldirishingiz mumkin.

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avraltod** [comment]
Hi,

Do you have any information on Mahalla data set?

Best, 

Avraa

**Reply by MIMAXUZ** [reply]
Yes, but they are not sorted. It is necessary to sort them out. There is no time for them. 
Regards.

**Reply by avraltod** [reply]
Thank you so much. 

Villages means bigger than Mahallas or it is the same for some cases. I know that there are over 9000 mahallas. 

Are these villages small cities (shaharcha? городских посёлков (гп), сельских сходов граждан (ссг))?

Best,

Avraa

**Reply by MIMAXUZ** [reply]
Yes.  There are over 12k Mahalla, not 9000.   names change very quickly.  That is why it is necessary to constantly update. Villages table is small towns (kichik sharchalar) and districts(tumanlar). In old versions, everything was in one table.  We did it separately in the new database.  There is also a list of neighborhoods, but they need to be sorted.  It takes an average of 1-3 months)))
Thanks.

**Reply by avraltod** [reply]
Thanks again. 

Is it possible to get shapefile for mahalla?

Can help with the cleaning and sorting?

Best,

Avraa

**Reply by MIMAXUZ** [reply]
Yes there is a general list.  But sorting requires an assistant.

**Reply by justinelliotmeyers** [reply]
any update on this list?

**Comment by mensenvau** [comment]
Thank you!!!! (Juda ham kerakli narsa bo'lipti buni domen qib API qilib qo'ysangiz ham bo'larkan.)

**Comment by Shohjahon** [comment]
_Proposal: Include SOATO Codes with Administrative Divisions of Uzbekistan_

Body:
Hello everyone,

I propose that we enhance the dataset by incorporating SOATO codes alongside the administrative divisions of Uzbekistan. Including these standardized codes is especially crucial during integration processes, as it ensures consistent and accurate data mapping between systems.

This update will not only improve data uniformity but also facilitate smoother interactions with external systems that rely on these classifications. I believe this change will add significant value to our project and make future integrations more efficient.

I look forward to hearing your thoughts and discussing this further.

Thank you!

**Reply by MIMAXUZ** [reply]
> _Proposal: Include SOATO Codes with Administrative Divisions of Uzbekistan_
> 
> Body: Hello everyone,
> 
> I propose that we enhance the dataset by incorporating SOATO codes alongside the administrative divisions of Uzbekistan. Including these standardized codes is especially crucial during integration processes, as it ensures consistent and accurate data mapping between systems.
> 
> This update will not only improve data uniformity but also facilitate smoother interactions with external systems that rely on these classifications. I believe this change will add significant value to our project and make future integrations more efficient.
> 
> I look forward to hearing your thoughts and discussing this further.
> 
> Thank you!

Yes, of course. We are currently working on this issue. We will release an update in the coming days that will include soato_id. The data is being checked for integrity. The data will be updated when everything is finished.
Regards.

**Reply by Shohjahon** [reply]
Here is possible soato codes of the regions, hope will be useful for further development
[СОАТO.zip](https://github.com/user-attachments/files/19650051/O.zip)

**Reply by Shohjahon** [reply]
[СОАТO.xlsx](https://github.com/user-attachments/files/19660480/O.xlsx)

**Reply by MIMAXUZ** [reply]
Thank you for your suggestions and solutions. We have updated the information after our long-term research.

**Comment by avraltod** [comment]
https://cs.egov.uz/#/tableData/65d3205dcb6031b39c078af7

Do you have any boundaries of these towns and villages?

**Reply by MIMAXUZ** [reply]
Not yet. We have indicated the source of the data in the wiki section. All received data will be separately examined and then released.
